pkg/service/dbaas/service.go

```diff
@@ -89,6 +108,9 @@ func (s *Service) fetchManagedDBaaS(ctx *Context) error {
 	for _, groupVersionResource := range groupVersionResources {
 		managedResources, err := s.k8sClient.Resource(groupVersionResource).List(ctx, metav1.ListOptions{})
 		if err != nil {
+			if err.Error() == "the server could not find the requested resource" {
+				continue
+			}
```

Comment:
As noted in the replaced PR I think we could use `erros.IsNotFound` here (might be wrong though, I usually use another k8s client)

Reply:
oh sry... I fixed it in the code an tested and forgot to push :)